### PR TITLE
chore: migrate to TheGraph Studio

### PR DESCRIPTION
**Pull request by alexandre-abrioux**
- update outdated libraries
- remove references to `goerli` and `rinkeby`
- migrate code and documentation to use Subgraph Studio instead of the deprecated hosted service

### CODE REVIEW

File: subgraph.mainnet.yaml [hunk 1]
```diff
@@ -26,9 +26,9 @@ dataSources:
     name: "Aggregator-0_2_0"
     network: "mainnet"
     source:
-      address: "0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA"
+      address: "0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9"
```
Comment (MantisClone) [inline subgraph.mainnet.yaml:29]: Why the change in address here?

File: subgraph.xdai.yaml [hunk 2]
```diff
@@ -22,3 +22,23 @@ dataSources:
         - event: AggregatorUpdated(address,address,address)
           handler: handleAggregatorUpdated
       file: ./src/aggregator.ts
+  - kind: ethereum/contract
+    name: "Aggregator-0_2_0"
+    network: "gnosis"
+    source:
+      address: "0xEEc4790306C43DC00cebbE4D0c36Fadf8634B533"
```
Comment (MantisClone) [inline subgraph.xdai.yaml:29]: Why do 0_1_0 and 0_2_0 have the same address?

**Review by MantisClone — APPROVED** [review]

File: subgraph.mainnet.yaml (hunk 1 above)
Comment (leoslr) [reply subgraph.mainnet.yaml:29]: `0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA` corresponds to the v2 of the `ChainlinkConversionPath` contract, but for mainnet it was never used because the v2 only fixed an issue for other networks.

But if in the index.json file that list the contract's addresses we put the v2 address, it will be targetted instead of the v1 when doing aggregators updates.

That's why the v2 address was replaced by the v1, and I guess that Alex's update automatically updated this.

**Review by leoslr — APPROVED** [review]

File: subgraph.mainnet.yaml (hunk 1 above)
Comment (alexandre-abrioux) [reply subgraph.mainnet.yaml:29]: Yes, this change was made automatically by `yarn prepare` after updating the `@requestnetwork/smart-contracts` package. The new contract version [0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA](https://etherscan.io/address/0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA) was deprecated, all aggregator updates are made on [0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9](https://etherscan.io/address/0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9), so both versions have the same address. See this PR: https://github.com/RequestNetwork/requestNetwork/pull/980
